serpentTools/parsers/sensitivity.py

```diff
@@ -401,8 +405,18 @@ def plot(self, resp, zai=None, pert=None, mat=None, sigma=3,
             label = labelFmt.format(r=resp, z=z, m=m, p=p)
             ax.errorbar(energies, yVals, yErrs, label=label,
                         drawstyle='steps-post')
+            if egrid is not None:
+                ax.vlines(egrid, min(yVals), max(yVals), colors='k',
+                          linestyles='dashed')
+
+        if xlabel is None:
+            if mevscale is False:
+                xlabel = 'Energy [eV]'
+            else:
+                xlabel = 'Energy [MeV]'
```

Comment:
Similar to the fix above
```suggestion
            xlabel = "Energy [MeV]" if mevscale else "Energy [eV]" 
```